cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -1022,11 +1022,14 @@ func WatchConfigMap() {
 	viper.SetConfigType("json")
 	viper.SetConfigName(autoReloadConfigFileName)
 	for {
-		verr := viper.ReadInConfig()
-		if verr == nil {
+		if verr := viper.ReadInConfig(); verr == nil {
 			break
 		} else {
-			glog.V(4).Infof("Can't read the autoreload config file %s/%s due to the error: %v, will retry in 3 seconds", autoReloadConfigFilePath, autoReloadConfigFileName, verr)
+			if _, ok := verr.(viper.ConfigFileNotFoundError); !ok {
+				glog.V(1).Infof("Can't read the autoreload config file %s/%s due to the error: %v, will retry in 3 seconds", autoReloadConfigFilePath, autoReloadConfigFileName, verr)
+			} else {
+				glog.V(4).Infof("Autoreload config file %s/%s not found", autoReloadConfigFilePath, autoReloadConfigFileName)
```

Comment:
The config file would be found here. This error message should indicate that the config was invalid.